lib/tmdb/networks/networks.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Tmdb.Networks do
+  use HTTPoison.Base
+
+  @doc ~S"""
+  Find networks by ID
+    ## Required Parameters
+    api_key
+    ## Optional Parameters
+    append_to_response:	Comma separated, any company method
+
+    ## Examples
+
+    iex> Tmdb.Companies.find(420)
+    %{"description" => nil, "headquarters" => nil, "homepage" => nil, "id" => 420,
+      "logo_path" => "/x1ynrDnk4RgbsnMI9yrdcdEm6Fu.png", "name" => "Marvel Studios",
+      "parent_company" => nil}
+ """
+  def find(id) do
+    get!("network/#{id}?").body
+  end
+
+  def tv(id) do
```

Comment:
Ok, but I was thinking you preferred to keep all endpoints separate and not build a search/discover as the endpoint have. I was putting all the find methods (Wanted to put Tmdb.Movies.imdb too and that was going to movies instead of discover/search in my head.

Anyway, I will delete the endpoint for now, so we can have cleaner code on my side before you submit v0.2, then you can let me know what would be the best way to approach the search/find/discover functions available. I was putting them on its own resources for simplifying/not having so much folders/endpoints available.